yarn.lock

```diff
@@ -1982,6 +1982,20 @@
     "@math.gl/core" "3.5.0-alpha.1"
     "@math.gl/geospatial" "3.5.0-alpha.1"
 
+"@loaders.gl/3d-tiles@^3.0.0-beta.5":
+  version "3.0.0-beta.5"
+  resolved "https://registry.yarnpkg.com/@loaders.gl/3d-tiles/-/3d-tiles-3.0.0-beta.5.tgz#8e3def3078107b18d7ae9d1c2bdb85cbf03ced73"
+  integrity sha512-i0eNVttpIsP/qG4OcUj7HQUd8U9prISudgyziBEFlpJYuwGa1lf5bL64ESBZ/xvR/4bTlDbb8VHpnkYfUykOVw==
+  dependencies:
+    "@loaders.gl/core" "3.0.0-beta.5"
+    "@loaders.gl/draco" "3.0.0-beta.5"
+    "@loaders.gl/gltf" "3.0.0-beta.5"
+    "@loaders.gl/loader-utils" "3.0.0-beta.5"
+    "@loaders.gl/math" "3.0.0-beta.5"
+    "@loaders.gl/tiles" "3.0.0-beta.5"
+    "@math.gl/core" "3.5.0-alpha.1"
+    "@math.gl/geospatial" "3.5.0-alpha.1"
+
 "@loaders.gl/core@3.0.0-alpha.21", "@loaders.gl/core@^3.0.0-alpha.21":
```

Comment:
Multiple loaders version is installed. Run `yarn list "@loaders.gl/*"` to make sure you have only one copy of each.

Reply:
Yeah. I now changed the version only in `geo` module. I wanted to do bumping in another PR.